lang/py/avro/schema.py

```diff
@@ -291,22 +331,24 @@ def has_name(self, name_attr, space_attr):
 
     def get_name(self, name_attr, space_attr):
         test = Name(name_attr, space_attr, self.default_namespace).fullname
-        if test not in self.names:
-            return None
-        return self.names[test]
+
+        return None if test not in self.names else self.names[test]
```

Comment:
Seems like `self.names.get(test)` should be equivalent.

Reply:
Changed in https://github.com/apache/avro/pull/1167/commits/3a250d8686b0b8e3098a1dc1da0be3839ecc3bb9